contributing/tests.md

```diff
@@ -7,268 +7,56 @@ describes them and gives pointers on how to run/debug them.
 
 The test types from most preferred to least preferred are:
 
-1. Unit Tests
-1. Vscode-integration Tests
-1. Integration Tests
-1. System Tests
+1. Unit Tests - jest: Found under the test/jest directory.
+   - `npm run test:unit`
+1. End to End Tests - _TDB_
 
-Ensure that you have set a default dev hub through the `sfdx force:auth:*` commands,
-passing in `--setdefaultdevhubusername`.
+**Deprecated**
 
-To run all tests, execute `npm run compile && npm run test` from the top-level
-folder.
+1. Unit Tests - mocha: Found under the test/unit directory. Not under active development.
+   - `npm run test:unit`
+1. vscode-integration - Tests that are somewhere between integration and unit tests. These are being phased out and there should be no new additions and minimual effort to
+   - Close all instances of vscode.
+   - `npm run test:integration`
+1. system-tests - These have been disabled for a long time and should be deleted.
```

Comment:
It feels a bit weird to say "should be deleted" here. Maybe "will soon be deleted".